### PR TITLE
HIVE-26970 : Fixes for some test failures with LLAP Driver

### PR DESCRIPTION
**Pull request by amanraj2520**
JIRA link : https://issues.apache.org/jira/browse/HIVE-26970

These tests were fixed already in branch-3.1 but were not backported to branch-3. Please follow this commit ID where these tests were fixed : a9dc9368

### CODE REVIEW

**Comment by amanraj2520** [comment]
@zabetak @abstractdog @nrg4878 Can you please approve this

File: testutils/ptest2/src/main/resources/log4j2.properties [hunk 1]
```diff
@@ -51,7 +51,7 @@ appender.FILE.strategy.max = 1
 loggers = Http, SpringFramework, OrgJclouds, Jclouds, Hive, NIOServerCnxn, ClientCnxnSocketNIO
 
 logger.Http.name = org.apache.http
-logger.Http.level = TRACE
+logger.Http.level = INFO
```
Comment (abstractdog) [inline testutils/ptest2/src/main/resources/log4j2.properties:54]: I don't don't think we need this anymore, ptest2 belongs to the old ptest framework

File: testutils/ptest2/src/test/resources/log4j2.properties [hunk 2]
```diff
@@ -36,7 +36,7 @@ appender.console.layout.pattern = %d{yy/MM/dd HH:mm:ss} %p %c{2}: %m%n
 loggers = Http, SpringFramework, OrgJclouds, Jclouds, Hive, NIOServerCnxn, ClientCnxnSocketNIO
 
 logger.Http.name = org.apache.http
-logger.Http.level = TRACE
+logger.Http.level = INFO
```
Comment (abstractdog) [inline testutils/ptest2/src/test/resources/log4j2.properties:39]: don't need this either

Comment (abstractdog) [reply testutils/ptest2/src/test/resources/log4j2.properties:39]: nevermind, we can simply copy the original commit a9dc9368149 as is

**Review by abstractdog — APPROVED** [review]

**Pull request merged**